### PR TITLE
feat: add github actions for FE

### PR DESCRIPTION
**Pull request by feng-tao**
### Summary of Changes

add pypi github action and pypi action . I didn't remove the travis as there seems to have more tests that python one.

### Tests

_What tests did you add or modify and why? If no tests were added or modified, explain why. Remove this line_

### Documentation

_What documentation did you add or modify and why? Add any relevant links then remove this line_

### CheckList
Make sure you have checked **all** steps below to ensure a timely review.
- [ ] PR title addresses the issue accurately and concisely. Example: "Updates the version of Flask to v1.0.2"
    - In case you are adding a dependency, check if the license complies with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
- [ ] PR includes a summary of changes, including screenshots of any UI changes. 
- [ ] PR adds unit tests, updates existing unit tests, __OR__ documents why no test additions or modifications are needed.
- [ ] In case of new functionality, my PR adds documentation that describes how to use it.
    - All the public python functions and the classes in the PR contain docstrings that explain what it does
- [ ] PR passes all tests documented in the [developer guide](https://github.com/lyft/amundsenfrontendlibrary/blob/master/docs/developer_guide.md#testing)

### CODE REVIEW

File: .github/workflows/pull_request.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      - name: Checkout
+        uses: actions/checkout@v1
+      - name: Setup python ${{ matrix.python-version }}
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: pip3 install -r requirements.txt && pip3 install .[all] && pip3 install codecov
+      - name: Run python unit tests
+        run: make test
```
Comment (Golodhros) [inline .github/workflows/pull_request.yml:30]: I think we miss here running the FE tests and a build.
I will check it tomorrow and add a suggestion!

Comment (Golodhros) [reply .github/workflows/pull_request.yml:30]: Maybe something like:
```
- name: Setup Frontend Application
  run: cd amundsen_application/static
- name: Install Frontend Dependencies
  run: npm install 
  run: npm install codecov -g
- name: Lint Frontend
  run: npm run lint
- name: Build Frontend
  run: npm run build
- name: Test Frontend
  run: npm run test
- name: Record Test Coverage for Frontend
  run: codecov
```

Comment (feng-tao) [reply .github/workflows/pull_request.yml:30]: thanks @Golodhros , pr updated.

**Comment by feng-tao** [comment]
@Golodhros @ttannis pr updated.

**Comment by feng-tao** [comment]
FE action is not triggered.

**Review by Golodhros — DISMISSED** [review]
Looks Great!

Thanks for this!

**Comment by feng-tao** [comment]
@Golodhros the FE action is not triggered :( still need to fix it.

**Comment by feng-tao** [comment]
Finally the FE action works...

**Comment by feng-tao** [comment]
Given @Golodhros already gave +1 before, will merge it now as the CI is green.

**Pull request merged**